themes/book/app/components/SiteFooter.tsx

```diff
@@ -0,0 +1,38 @@
+import { MastodonIcon, TwitterIcon, GithubIcon, DiscordIcon } from '@scienceicons/react/24/solid';
+import { Link } from '@remix-run/react';
+import classNames from 'classnames';
+import { extractPart } from 'myst-common';
+import { MyST } from 'myst-to-react';
+import { select, selectAll, matches } from 'unist-util-select';
+import { filter } from 'unist-util-filter';
+
+
+export function SiteFooter({ tight, content }: { tight?: boolean }) {
+  const footerBlock = selectAll('block', content).find((node) => node.kind == 'footer');
+  const footerColumnsBlock = selectAll('block', content).find((node) => node.kind == 'footer-columns');
+  const footerLists = selectAll('list', footerColumnsBlock);
+
+  return (
+    <section className={classNames('col-screen article-grid', { 'mt-10': !tight, 'mt-0': tight })}>
```

Comment:
nit: use conditional, add horizontal padding

```suggestion
    <section className={classNames('col-screen article-grid px-4', tight ? 'mt-0' : 'mt-10')}>
```